core/import_data/import_replenishments.py

```diff
@@ -0,0 +1,127 @@
+import decimal
+import logging
+from decimal import Decimal
+
+import pandas as pd
+
+from core.models import Replenishment, Country, Contribution
+from core.import_data.utils import (
+    IMPORT_RESOURCES_DIR,
+    delete_old_data,
+)
+
+logger = logging.getLogger(__name__)
+
+COUNTRY_MAPPING = {
+    "Czech Republic": "Czechia",
+    "Slovak Republic": "Slovakia",
+    "Netherlands": "Netherlands (Kingdom of the)",
+    "United Kingdom": "United Kingdom of Great Britain and Northern Ireland",
+}
+
+REPLENISHMENT_YEARS = [(2021, 2023), (2024, 2026)]
+COUNTRY_COLUMN = 0
+UN_SCALE_OF_ASSESSMENT_COLUMN = 1
+ADJUSTED_SCALE_OF_ASSESSMENT_COLUMN = 2
+CONTRIBUTION_COLUMN = 3
+AVERAGE_INFLATION_RATE_COLUMN = 4
+QUALIFIES_FOR_FIXED_RATE_MECHANISM_COLUMN = 5
+FIXED_EXCHANGE_RATE_MECHANISM_COLUMN = 6
+FIXED_EXCHANGE_RATE_CURRENCY_COLUMN = 7
+FIXED_EXCHANGE_RATE_NATIONAL_CURRENCY_COLUMN = 8
+
+
+def decimal_converter(value):
+    try:
+        return Decimal(str(value))
```

Comment:
Unfortunately, Excel has some unsolved issues with [floating point precision] (https://techcommunity.microsoft.com/t5/excel/really-weird-decimal-amounts/m-p/143304).For example, if you try to enter 0.1 in Excel, it might be stored internally as 0.10000000000000000555.

If the files you use for import are susceptible to this issue, I suggest using the get_decimal_from_excel_string method from the utils file to ensure the data remains accurate and clean.

Reply:
> if you try to enter 0.1 in Excel, it might be stored internally as 0.10000000000000000555

Yeah, because Excel stores numbers as 64bit floats and assures only 15 digits of correctness if I'm not mistaken. It's not an issue, that's how floats work. The same thing happens in Python with floats

```
>>> 0.1 + 0.2
0.30000000000000004
```

The thing is, we're working with money and I want to make sure the results are very close to the Excel files even when using formulas.